### DISCUSSION TITLE
Ranking number in small profile overlaps each others

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#28101] [Q&A] [answered]
![image](https://github.com/ppy/osu/assets/59832159/0463e10b-34d1-4757-9556-0660ca15d0f9)  
in the image `Global Ranking` and `Country Ranking` number is being overlapped on each other (a little bit) in small profile thing that you can click to get to actual profile window

**Comment by Joehuu** [comment, marked as answer]
This is already tracked as an issue in https://github.com/ppy/osu/issues/27296.
